subprojects/testing-jvm/src/testFixtures/groovy/org/gradle/testing/fixture/JUnitPlatformTestFixture.groovy

```diff
@@ -108,6 +109,11 @@ trait JUnitPlatformTestFixture {
             return this
         }
 
+        TestClass withAdditionalContent(String content) {
```

Comment:
TBH I'm not sure I'm in love with the whole `testClass` and `testMethod` structure in this test, since in order to understand the structure of the test classes you need to learn a new niche DSL. Obviously not something to fix in here, but I was able to understand the tests added in the other classes more easily than the ones for junit platform.

Reply:
> but I was able to understand the tests added in the other classes more easily than the ones for junit platform.

👍

Reply:
I think you're right.  I almost yanked it out before I cut the PR, but then for some reason I didn't.  It can be expressive when you've got a bunch of simple tests that pass/fail/skip without a bunch of boilerplate, but here it doesn't seem to simplify anything.  

I've changed it to just a raw test class.